### ISSUE TITLE
react-native-group-checkbox is not working

### ISSUE THREAD

**Issue by MuhammadSaadTabani** [tdle94/react-native-group-checkbox#1]
Unable to resolve module `AccessibilityInfo` from `node_modules\react-native-group-checkbox\node_modules\react-native\Libraries\react-native\react-native-implementation.js`: AccessibilityInfo could not be found within the project.

**Comment by tdle94** [comment]
thanks, i will look into it
